File: cpp/pe001.cpp
#include <stdio.h>

int calculatesum(int max_value) {
    int the_sum = 0;
    for(int i = 3; i < max_value; i++) {
        if(i%3==0 || i%5==0) {
            the_sum += i;
        }
    }

    return the_sum;
}
int main(int argc, char const *argv[]) {
    int max_value = 1000;
    int the_sum = calculatesum(max_value);
    printf("The sum is: %d\n", the_sum); //233168
    return 0;
}

File: python/pe001.py
def sumMultiples(max_value):
    thesum = 0

    for i in range(max_value):
        if i%3==0 or i%5==0:
            thesum += i

    return thesum

if __name__=="__main__":
    max_value = 1000
    thesum = sumMultiples(max_value)
    print(thesum) # 233168
